### PR TITLE
Clarifying usage of delivery_whitelist

### PR DESCRIPTION
**Pull request by tomhv**
There was nothing to suggest that `delivery_whitelist` could not be used on its own. We had to find this out by trial an error.

Hopefully, this change (or better equivalent) will help other people.

Cheers.

### CODE REVIEW

**Review by wouterj — APPROVED** [review]
Looks fine by me!

status: reviewed

**Review by marcverney — CHANGES_REQUESTED** [review]

File: email/dev_environment.rst [hunk 1]
```diff
@@ -190,6 +190,10 @@ In the above example all email messages will be redirected to ``dev@example.com`
 and messages sent to the ``admin@mydomain.com`` address or to any email address
 belonging to the domain ``specialdomain.com`` will also be delivered as normal.
 
+.. caution::
+
+    The ``delivery_whitelist`` option can only be used in conjuction with the ``delivery_addresses`` option.
```
[the remark this answers is not in the document]
Comment (javiereguiluz) [reply email/dev_environment.rst]: Can we reword this to make it easier to read? Something like this:

```
The ``delivery_whitelist`` option is ignored when the ``delivery_addresses``
option is not defined.
```

[the remark this answers is not in the document]
Comment (tomhv) [reply email/dev_environment.rst]: What about this:

The ``delivery_whitelist`` option is ignored unless the ``delivery_addresses`` option is defined.

[the remark this answers is not in the document]
Comment (javiereguiluz) [reply email/dev_environment.rst]: Nice and concise! I like it. Thanks!

**Comment by tomhv** [comment]
I need to squash these commits into a single commit, right?

**Comment by javiereguiluz** [comment]
@tomhv there's no need because we use a tool that squashes everything automatically. Cheers!

**Comment by weaverryan** [comment]
Thank you @tomhv!
